doc/conf.py

```diff
@@ -4,16 +4,25 @@
 #
 # This code is part of the Fatiando a Terra project (https://www.fatiando.org)
 #
-# -*- coding: utf-8 -*-
-import sys
 import os
 import datetime
-import sphinx_rtd_theme
 import sphinx_gallery
-from sphinx_gallery.sorting import FileNameSortKey
+from sphinx_gallery.sorting import ExampleTitleSortKey
 
 from harmonica.version import full_version
 
+
+# Project information
+# -----------------------------------------------------------------------------
+project = "Harmonica"
+copyright = f"{datetime.date.today().year}, The {project} Developers"
```

Comment:
```suggestion
copyright = f"2018-{datetime.date.today().year}, The {project} Developers"
```

Ah, right. Sorry forgot about that. Fixed!